test/e2e/networking.go

```diff
@@ -94,29 +96,18 @@ var _ = Describe("Networking", func() {
 			Failf("Unexpected error code, expected 200, got, %v (%v)", resp.StatusCode, resp)
 		}
 
-		By("Building a namespace api object")
-		var ns = svcname + "-" + randomSuffix()
-
-		namespaceObj = &api.Namespace{
-			ObjectMeta: api.ObjectMeta{
-				Name:      ns,
-				Namespace: "",
-			},
-			Status: api.NamespaceStatus{},
-		}
-
 		By("Creating a kubernetes client")
 		c, err = loadClient()
 		Expect(err).NotTo(HaveOccurred())
 
-		By("Creating a namespace for this test suite")
-		_, err = c.Namespaces().Create(namespaceObj)
+		By("Building a namespace api object")
+		namespaceObj, err = createGinkgoNS("nettest", c)
 		Expect(err).NotTo(HaveOccurred())
 
 	})
 
 	AfterEach(func() {
-		By("Destroying namespace for this suite")
+		By(fmt.Sprintf("Destroying namespace for this suite %v", namespaceObj.Name))
```

Comment:
We should consider moving most of the namespace creation stuff to BeforeSuite() and the namespace deletion to AfterSuite(), rather than duplicate it in every test.